src/api/apiFetch.js

```diff
@@ -10,9 +10,11 @@ import { makeErrorFromApi } from './apiErrors';
 
 const apiVersion = 'api/v1';
 
-export const getFetchParams = <P>(auth: Auth, params: P): { headers: mixed, ...P } => {
-  // $FlowFixMe This is purely a no-op, and Flow even knows that. :-/
-  const { body } = (params: { body?: mixed });
+export const getFetchParams = <P: $Rest<$Exact<RequestOptions>, {| headers: mixed |}>>(
```

Comment:
`$Rest` is very similar to `$Diff`, and rereading what the difference (ha) is between them, I think `$Diff` is more on point here:
https://flow.org/en/docs/types/utilities/#toc-rest
because it's more purely "this object type, just without these particular properties".